.azurepipelines/python_templates/flake8-test-steps.yml

```diff
@@ -0,0 +1,27 @@
+# File flake8-test-steps.yml
+#
+# template file to run flake8 and if error publish log
+#
+# Copyright (c) 2019, Microsoft Corporation
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+parameters:
+  none: ''
+  
+steps:
+- script: flake8 .
+  displayName: 'Run flake8'
+  condition: succeededOrFailed()
+
+# Only capture and archive the lint log on failures.
+- script: flake8 . > flake8.err.log
+  displayName: 'Capture flake8 failures'
+  condition: Failed()
+
+- task: PublishBuildArtifacts@1
+  inputs:
+    pathtoPublish: 'flake8.err.log' 
+    artifactName: 'Flake8 Error log file'
+  continueOnError: true
+  condition: Failed()
```

Comment:
nit, newline

Reply:
There are a few more places this happens too

Reply:
will update, thanks!

Reply:
Updated.